bin/gen_PKGBUILD.py

```diff
@@ -0,0 +1,44 @@
+import tomli
+
+with open("pyproject.toml", 'rb') as f:
+    pyproject = tomli.load(f)
+
+description = pyproject["project"]["description"]
+version = open('anesthetic/_version.py','r').readlines()[0].split("=")[1].strip().strip("'")
+url = pyproject["project"]["urls"]["Homepage"]
+pyproject["project"]["dependencies"]
+rel=2
+
+
+PKGBUILD = """# Maintainer: Will Handley <wh260@cam.ac.uk> (aur.archlinux.org/account/wjhandley)
+pkgname=python-anesthetic
+_name=${pkgname#python-}
+pkgver=%s
+pkgrel=%s
+pkgdesc="%s"
+arch=(any)
+url="%s"
+license=(MIT)
+groups=()
+depends=(python-numpy python-matplotlib python-scipy python-pandas)
+makedepends=(python-build python-installer)
+provides=()
```

Comment:
Might be worth adding `anesthetic`. Someone (wink wink) might provide a C++ GUI for anesthetic, and a Rust API-compatible implementation.

Reply:
Done